### PR TITLE
[SQL-DS-CACHE-136]Update the documents for OAP 1.1.1

### PR DESCRIPTION
**Pull request by HongW2019**
## What changes were proposed in this pull request?

- [x] Remove the NUMA related info
- [x] Specify the Arrow version to 4.0.0 in Developer Guide
- [x] Update the Changelog and details for OAP 

## How was this patch tested?

N/A

### CODE REVIEW

File: docs/Advanced-Configuration.md [hunk 1]
```diff
@@ -50,21 +48,9 @@ The following are required to configure OAP to use PMem cache.
...
-   
    For more information you can refer to [Quick Start Guide: Provision Intel® Optane™ DC Persistent Memory](https://software.intel.com/content/www/us/en/develop/articles/quick-start-guide-configure-intel-optane-dc-persistent-memory-on-linux.html)
 
-#### Configuration for NUMA
 
-1. Install `numactl` to bind the executor to the PMem device on the same NUMA node. 
-
-   ```yum install numactl -y ```
-
-2. We strongly recommend you use NUMA-patched Spark to achieve better performance gain for the following 3 cache strategies. Besides, currently using Community Spark occasionally has the problem of two executors being bound to the same PMem path.
```
Comment (HongW2019) [inline docs/Advanced-Configuration.md:63]: @winningsix Please take a review for removing NUMA-related info, thanks!  and we still need more modifications on the pmem path config of 'persistent-memory.xml'.

**Comment by HongW2019** [comment]
@yma11 Please take a review, thanks a lot!

File: docs/Developer-Guide.md [hunk 2]
```diff
@@ -81,15 +78,15 @@ To use optimized Plasma cache with OAP, you need following components:
...
+   (3) `arrow-plasma-4.0.0.jar`: will be used when compile oap and spark runtime also need it. 
 
 - `.so` file and binary file  
   Clone code from Arrow repo and run following commands, this will install `libplasma.so`, `libarrow.so`, `libplasma_java.so` and `plasma-store-server` to your system path(`/usr/lib64` by default). And if you are using Spark in a cluster environment, you can copy these files to all nodes in your cluster if the OS or distribution are same, otherwise, you need compile it on each node.
   
 ```
 cd /tmp
 git clone https://github.com/oap-project/arrow.git
-cd arrow && git checkout arrow-3.0.0-oap
+cd arrow && git checkout arrow-4.0.0-oap-1.1.1
```
Comment (Aaxida) [inline docs/Developer-Guide.md:39]: Has confirmed with @jikunshang that the 4.0.0 version is workable

File: docs/Developer-Guide.md [hunk 3]
```diff
@@ -135,26 +132,4 @@ cd <path>/sql-ds-cache
 mvn clean -DskipTests package
 ```
```
Comment (Aaxida) [inline docs/Developer-Guide.md:79]: It might be better to add 'guava/vmem is not recommended for the current release'

Comment (HongW2019) [reply docs/Developer-Guide.md:79]: Yes, I forgot to update this part. As for we have removed the rest options in User Guide, we'd better remove these guava/vmem info in Developer Guide for current minor release, right?

Comment (yma11) [reply docs/Developer-Guide.md:79]: we may not remove guava/vmem in current release but in guava/vmem section, tell user need to apply numa patch if they want to use it.

Comment (HongW2019) [reply docs/Developer-Guide.md:79]: Yes, it also makes sense,  although this PR is followed by the emails decision. If not remove, please help open a new PR to upload a NUMA patch for Spark-3.1.1 and update the related docs.

**Comment by HongW2019** [comment]
@yma11 @Aaxida Any suggestions to this PR,  are we going to abort this PR or merge it first then submit new PRs if more modifications needed?

**Comment by yma11** [comment]
We can merge this PR first. And @Aaxida may add the numa information for vmem/guava cache later.

**Comment by HongW2019** [comment]
> We can merge this PR first. And @Aaxida may add the numa information for vmem/guava cache later.

Got it.  @zhixingheyi-tian Please help merge it, thanks.

**Pull request merged**